doc/quickstart/configure.rst

```diff
@@ -46,85 +46,80 @@ with explanations in a commented line above each option:
-  # Set to null to use the number of available CPUs.
-  # If you run out of memory, try setting max_parallel_tasks to 1 and check the
-  # amount of memory you need for that by inspecting the file
-  # run/resource_usage.txt in the output directory. Using the number there you
-  # can increase the number of parallel tasks again to a reasonable number for
-  # the amount of memory available in your system.
-  max_parallel_tasks: null
-
-  # Set the console log level debug, [info], warning, error
-  # for much more information printed to screen set log_level: debug
-  log_level: info
-
-  # Exit on warning (only for NCL diagnostic scripts)? true/[false]
-  exit_on_warning: false
-
-  # Plot file format? [png]/pdf/ps/eps/epsi
-  output_file_type: png
-
-  # Remove the ``preproc`` dir if the run was successful
-  # By default this option is set to "true", so all preprocessor output files
-  # will be removed after a successful run. Set to "false" if you need those files.
-  remove_preproc_dir: true
-
-  # Use netCDF compression true/[false]
-  compress_netcdf: false
-
-  # Save intermediary cubes in the preprocessor true/[false]
-  # set to true will save the output cube from each preprocessing step
-  # these files are numbered according to the preprocessing order
-  save_intermediary_cubes: false
-
-  # Use a profiling tool for the diagnostic run [false]/true
-  # A profiler tells you which functions in your code take most time to run.
-  # For this purpose we use vprof, see below for notes
-  # Only available for Python diagnostics
-  profile_diagnostic: false
-
-  # Path to custom config-developer file, to customise project configurations.
-  # See config-developer.yml for an example. Set to "null" to use the default
-  config_developer_file: null
-
-.. code-block:: yaml
-
-  # Auxiliary data directory (used for some additional datasets)
-  auxiliary_data_dir: ~/auxiliary_data
+   # Directory where results are stored
+   output_dir: ~/esmvaltool_output
+
+   # Directory for storing downloaded climate data
+   download_dir: ~/climate_data
+
+   # Auxiliary data directory (used for some additional datasets)
+   auxiliary_data_dir: ~/auxiliary_data
+
+   # Run at most this many tasks in parallel [null]/1/2/3/4/..
+   # Set to null to use the number of available CPUs.
+   # If you run out of memory, try setting max_parallel_tasks to 1 and check the
+   # amount of memory you need for that by inspecting the file
+   # run/resource_usage.txt in the output directory. Using the number there you
+   # can increase the number of parallel tasks again to a reasonable number for
+   # the amount of memory available in your system.
+   max_parallel_tasks: null
+
+   # Disable the automatic download of missing CMIP3, CMIP5, CMIP6, CORDEX,
+   # and obs4MIPs data from ESGF by default. This is useful if you are working
+   # on a computer without an internet connection.
+   offline: true
+
+   # Log level of the console
+   log_level: info
```

Comment:
It would be nice to have the available options listed here, with square brackets around the default, as was previously the case